src/write.c

```diff
@@ -538,6 +538,10 @@ static avifBool avifEncoderDetectChanges(const avifEncoder * encoder, avifEncode
 }
 
 // Subset of avifEncoderWriteColorProperties() for the properties clli, pasp, clap, irot, imir.
+// outputStream is the final bitstream that will be output by the libavif encoder API.
+// dedupStream is either equal to outputStream or to dedup->s which is a temporary stream used
```

Comment:
Nit: Add `&` before `dedup->s`.

Optional: It might be good to move the new comment closer to `avifEncoderWriteColorProperties()`, where these variables/parameters are actually used.